### PR TITLE
feat: add mete teams to our teams section

### PR DESCRIPTION
**Pull request by paulstey**
This is a small PR that adds Mete's teams to the `Our Teams` section.

### CODE REVIEW

File: about/teams.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-4
 Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-499a-a1f3-5f9b24c390e4)
 >Our software engineers help researchers to get the most out of high-performance computing (HPC) resources. To achieve this, we partner with users to design, debug, and optimize their code to run in an HPC environment. We also collaborate with new and non-traditional HPC users to help them to scale up and accelerate their science.
 
+### High-Performance Computing Systems
```
Comment (mcmcgrath13) [inline about/teams.md:14]: the name between this and HPC are very close - should the arc team be renamed?

File: about/teams.md [hunk 2]
```diff
@@ -11,6 +11,14 @@ Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-4
...
+Leadership: [Mete Tunca](https://directory.brown.edu/uuid/d0e7b3b6-f6b4-f28b-3bfd-08bccf05ac31)
+>Our HPC systems engineers are responsible for building and maintaining the hardware and system-level software for Brown's primary supercomputer, Oscar. This includes designing the overall architecture of the system, the storage and networking configurations, as well as the operating system and the system libraries. We also work with researcher to plan hardware purchases for grant applications.
```
Comment (mcmcgrath13) [inline about/teams.md:16]:
```suggestion
>Our HPC systems engineers are responsible for building and maintaining the hardware and system-level software for Brown's primary supercomputer, Oscar. This includes designing the overall architecture of the system, the storage and networking configurations, as well as the operating system and the system libraries. We also work with researchers to plan hardware purchases for grant applications. 
```

File: about/teams.md [hunk 3]
```diff
@@ -11,6 +11,14 @@ Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-4
...
+
+### Research Technical Services
+Leadership: [Mete Tunca](https://directory.brown.edu/uuid/d0e7b3b6-f6b4-f28b-3bfd-08bccf05ac31)
+>Our team research computing consultants support researchers across all of our research computing environments and tools. This includes on Oscar, Stronghold, Globus, Hibernate, JupyterHub, and others. We can help with onboarding to our systems, we can help with debugging, and we can help you decide what systems are best suited to your research computing needs. We hold regular weekly office hours, and are happy to help with tutorials and additional consultation sessions outside of office hours.
```
Comment (mcmcgrath13) [inline about/teams.md:24]:
```suggestion
>Our team of research computing consultants support researchers across all of our research computing environments and tools. This includes Oscar, Stronghold, Globus, Hibernate, JupyterHub, and others. We can help with onboarding to our systems, debugging, and deciding what systems are best suited to your research computing needs. We hold regular weekly office hours, and are happy to help with tutorials and additional consultation sessions outside of office hours. 
```

File: about/teams.md (hunk 1 above)
Comment (paulstey) [reply about/teams.md:14]: Yeah, agreed, it's not ideal... but I don't see much alternative. Those are the team's names. ¯\_(ツ)_/¯

**Comment by paulstey** [comment]
Thank you so much, @mcmcgrath13!!

**Review by mcmcgrath13 — COMMENTED** [review]
what's the ordering of the teams? at first I was thinking alpha, but then I remembered `g` comes before `h`... or should there be meta groups of arc vs research services at the `##` level?

**Review by mcmcgrath13 — APPROVED** [review]
🫖

**Comment by paulstey** [comment]
Thank you so much, @mcmcgrath13!!

**Pull request merged**